web/pandas/pdeps/00xx-string-dtype.md

```diff
@@ -0,0 +1,253 @@
+   significant part of the user base that has PyArrow installed) in performance.
+2. In case we eventually transition to use `pd.NA` as the default missing value
+   sentinel, we will need a migration path for _all_ our data types, and thus
+   the challenges around this will not be unique to the string dtype and
+   therefore not a reason to delay this.
+
+### Why not use the existing StringDtype with `pd.NA`?
+
+Wouldn't adding even more variants of the string dtype will make things only more
+confusing? Indeed, this proposal unfortunately introduces more variants of the
+string dtype. However, the reason for this is to ensure the actual default user
+experience is _less_ confusing, and the new string dtype fits better with the
+other default data types.
+
+If the new default string data type would use `pd.NA`, then after some
+operations, a user can easily end up with a DataFrame that mixes columns using
+`NaN` semantics and columns using `NA` semantics (and thus a DataFrame that
+could have columns with two different int64, two different float64, two different
+bool, etc dtypes). This would lead to a very confusing default experience.
+
+With the proposed new variant of the StringDtype, this will ensure that for the
+_default_ experience, a user will only see only 1 kind of integer dtype, only
+kind of 1 bool dtype, etc. For now, a user should only get columns with an
+`ArrowDtype` and/or using `pd.NA` when explicitly opting into this.
+
+## Backward compatibility
+
+The most visible backwards incompatible change will be that columns with string
+data will no longer have an `object` dtype. Therefore, code that assumes
+`object` dtype (such as `ser.dtype == object`) will need to be updated.
+
+To allow testing your code in advance, the
+`pd.options.future.infer_string = True` option is available.
+
+Otherwise, the actual string-specific functionality (such as the `.str` accessor
+methods) should all keep working as is. By preserving the current missing value
+semantics, this proposal is also backwards compatible on this aspect.
+
+One other backwards incompatible change is present for early adopters of the
+existing `StringDtype`. In pandas 3.0, calling `pd.StringDtype()` will start
+returning the new default string dtype, while up to now this returned the
+experimental string dtype using `pd.NA` introduced in pandas 1.0. Those users
+will need to start specifying a keyword in the dtype constructor if they want to
+keep using `pd.NA` (but if they just want to have a dedicated string dtype, they
+don't need to change their code).
+
+## Timeline
+
+The future PyArrow-backed string dtype was already made available behind a feature
+flag in pandas 2.1 (by `pd.options.future.infer_string = True`).
+
+Some small enhancements or fixes (or naming changes) might still be needed and
+can be backported to pandas 2.2.x.
+
+The variant using numpy object-dtype could potentially also be backported to
+2.2.x to allow easier testing.
```

Comment:
Don't you mean "numpy 2.0 string-dtype" ?  Because the "numpy object-dtype" is currently there?  This labeling is confusing.

Given the discussions elsewhere about where the names are now for the dtype `"string[pyarrow]"`, `"string[pyarrow_numpy]"`, etc. which I can't keep track of, I think that the nomenclature in terms of strings that work should be specified, comparing what is in 2.2 to what would be implemented as a result of this PDEP.

The possible strings are confusing.  Which strings can be used in a `dtype` argument in a constructor or `astype()` ?  Which strings would be seen by users when they do `Series.dtype` ?  There is what currently exists in pandas 2.2, and what would exist based on this PDEP, and I'm not seeing what the conclusion of that would be.